docs/tutorials/runtime.md

```diff
@@ -0,0 +1,265 @@
+# Tutorial 7: Customize Runtime Settings
+
+In this tutorial, we will introduce some methods about how to customize optimization methods, training schedules, workflow and hooks when running your own settings for the project.
+
+<!-- TOC -->
+
+- [Customize Workflow](#customize-workflow)
+- [Hooks](#hooks)
+  - [Default runtime hooks](#default-runtime-hooks)
+    - [CheckpointSaverHook](#checkpointsaverhook)
+    - [LoggerHooks](#loggerhooks)
+    - [EvaluationHook](#evaluationhook)
+  - [Use implemented hooks](#use-implemented-hooks)
+  - [Customize self-implemented hooks](#customize-self-implemented-hooks)
+    - [1. Implement a new hook](#1-implement-a-new-hook)
+    - [2. Register the new hook](#2-register-the-new-hook)
+    - [3. Modify the config](#3-modify-the-config)
+- [FAQ](#faq)
+
+<!-- TOC -->
+
+## Customize Workflow
+
+By default, we recommend users to use **`EvaluationHook`** to do evaluation after training epoch, but they can still use `val` workflow as an alternative.
```

Comment:
This line should be moved after the introduction.
And consider adding a note to remind users that modifying workflow is unnecessary in most situations.